### DISCUSSION TITLE
Tasks for CG Participants

### DISCUSSION THREAD

**Discussion by egekorkan** [w3c-cg/wot-cg#43] [Organisational]
In today's internal meeting, we had some brainstorming about what CG participants can do as a CG activity. These should be work items that each can contribute to. So far the ideas are:
1. Meetup Organization: Finding new presenters, understanding their idea/presentation, establishing outreach of the presentation, creating and doing speaker introductions, and moderating the session.
2. Creating a weekly (or biweekly, monthly) "digest" of WoT: This means summarizing discussions and news about WoT, which can be: new additions to the specifications or interesting discussions that have happened/happening on GitHub or Discord, summarizing a meetup when there is one.
3. Contributing to the "What is WoT?" tutorial series: This includes helping with the creation of animations, writing script text, coming up with new sections to the tutorial or simply giving feedback on the process
4. Writing Binding Documents: This is not part of our charter yet but with the new structure of the next WoT deliverables of the working group, it can possible to host some of the binding document creation at the community group. This means that CG participants can write a new protocol binding, e.g. AMQP, Matter etc.

We want to kickstart this discussion and ask for new ideas. After getting a final list, we will create a poll to understand where CG participants want to contribute to.

**Comment by FadySalama** [comment]
Another point could be tooling: Help with creating, maintaining, and cataloging tools.

**Comment by relu91** [comment]
Expanding point 2 with even more granular subtasks:
- choose a publishing platform
- depending on the platform develop some branding style
- report structure
- writing the actual content (sources: public minutes, closed PRs, closed issues, mailing list).
